src/test/shell/bazel/bazel_java_tools_dist_test.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+#
+# Copyright 2019 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Would you mind adding a comment on what this test is supposed to test and what differentiates it from bazel_java_tools_test.sh? (I don't know what 'dist' means in this context, so maybe other people will not know too).

Reply:
Good idea, thanks!